clients/node/src/errorTypes.ts

```diff
@@ -2,7 +2,7 @@
  * Structure representing an error response from the api, wrapped in SanitisedError.
  *
  * Defined in uk.gov.justice.digital.hmpps.hmppsnonassociationsapi.config.ErrorResponse class
- * https://github.com/ministryofjustice/hmpps-non-associations-api/blob/f6002aa1da50b8c4ccd3613e970327d5c67c44ae/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/config/HmppsNonAssociationsApiExceptionHandler.kt#L240-L261
+ * see https://github.com/ministryofjustice/hmpps-non-associations-api
```

Comment:
I kinda liked the direct link to the code.
Although sure code change over time, pin the commit hash is an idea but then it may be stale code...so maybe that's why you made the link more generic?

Reply:
problem is i'm changing `ErrorCode` here so cannot link to the un-merged code… so i thought i might give up on these direct links. what do you think is better 1) drop all direct commit links 2) link to `main` and cross fingers that if something moves, we remember to repoint these links? 3) wait until merge to put in direct commit links

Reply:
I think this is probably the best compromise: Comment has class path/packages and link is to repo.